pkgs/development/mobile/androidenv/compose-android-packages.nix

```diff
@@ -293,27 +311,22 @@ rec {
 
       # Expose common executables in bin/
       mkdir -p $out/bin
-      find $PWD/tools -not -path '*/\.*' -type f -executable -mindepth 1 -maxdepth 1 | while read i
+
+      for i in ${platform-tools}/bin/*
```

Comment:
for i in ${platform-tools}/bin/*; do

Reply:
Fixed on [41acb57](https://github.com/NixOS/nixpkgs/pull/208793/commits/41acb572e21fd2d1776a0e148d059a2188d68dad)